napi/oxlint2/src-js/index.js

```diff
@@ -123,6 +121,14 @@ class Context {
   getPhysicalFilename() {
     return this.physicalFilename;
   }
+
+  static {
+    setupContextForFile = (context, ruleIndex, filePath) => {
+      context.#ruleIndex = ruleIndex;
+      context.filename = filePath;
+      context.physicalFilename = filePath;
+    };
+  }
```

Comment:
out of interest is static faster than a non-static method directly on the class??

Reply:
lgtm